utils.go

```diff
@@ -320,7 +320,14 @@ func getCgroupDestination(subsystem string) (string, error) {
 	defer f.Close()
 	s := bufio.NewScanner(f)
 	for s.Scan() {
-		fields := strings.Fields(s.Text())
+		fields := strings.Split(s.Text(), " ")
+		if len(fields) < 10 {
+			// broken mountinfo?
+			continue
+		}
+		if fields[len(fields)-3] != "cgroup" {
```

Comment:
This assumes we do not use this function for cgroupv2 mount (as far as I can see we're not, since the v2 unified mount point does not list individual controllers in mount options).